cosmos/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.10.0"
```

Comment:
Since we changed the approach, does this change still make sense? What are the benefits of having the version here and not in `cosmos/__init__.py`?

Reply:
yes, nice catch. Moved it back to init module.